src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -898,11 +898,13 @@ void goto_instrument_parse_optionst::get_goto_program()
 {
   status() << "Reading GOTO program from `" << cmdline.args[0] << "'" << eom;
 
+  config.set(cmdline);
+
```

Comment:
Is this actually necessary/useful at all? AFAIK the config will be overwritten below?

Reply:
Yes, but not all of it -- only what is called 'architectural configuration'. Say include paths aren't.